joomla/joomla-cms
# Change the defaults for new installs to disable com_mailto in articles

### Summary of Changes

Change the defaults for new installs to disable com_mailto in articles.

As discussed with the JSST this is now a public tracker item: cc @SniperSister @mbabker

### Testing Instructions

- Install [this branch](https://github.com/zero-24/joomla-cms/archive/mailto_disabled_bydefault.zip)
- make sure the default for com_mailto settings in com_content Global configuration is off

### Expected result

default is off

### Actual result

default is on

### Documentation Changes Required

none

### Additional notes

When this got merged I'm going to build a PR against 4.0 that set the correct settings to the xml and PHP Code too if needed.
